thirdeye/thirdeye-frontend/app/utils/constants.js

```diff
@@ -10,88 +10,73 @@ export default {
+
+# Give a name for this anomaly detection pipeline (should be unique)
 detectionName: name_of_the_detection
-# the metric to detect the anomalies
-metric: metric_name
-# the data set name for this metric
-dataset: dataset_name
-# ThirdEye pipeline type. Just fill in Composite
-pipelineType: Composite
 
-# (Optional) Config dimension exploration
-dimensionExploration:
-  # Create an alert for each dimension value in the dimension
-  dimensions:
-    - dimensionName
+# Tell the alert recipients what it means if this alert is fired
+description: If this alert fires then it means so-and-so and check so-and-so for irregularities
 
-  # (Optional) only create alert for the dimension value if the contribution to
-  # overall metric is above the threshold
-  minContribution: 0.05
+# The metric you want to do anomaly detection on. You may type a few characters and look ahead (ctrl + space) to auto-fill
+metric: metric_name
 
-  # (Optional) only create alert for the top k dimension values
-  k: 10
+# The dataset or UMP table name to which the metric belongs. Look ahead should auto populate this field
+dataset: dataset_name
 
-# (Optional) Filter the metric by
-filters:
-  dimensionName1:
-     - dimensionValue1
-     - dimensionValue2
-  dimensionName2:
-     - dimensionValue3
+# ThirdEye pipeline type. Only "Composite" is supported now.
+pipelineType: Composite
 
-# configure rules of anomaly detection
+# Configure multiple rules. ThirdEye supports single or a list of rules combined together with "OR" relationship
 rules:
-# configure the first rule
-- # configure the detection rule. ThirdEye will detect anomalies based on the
-  # detection rules.
-  detection:
-      # give a name for the detection rule
-    - name: detection_rule_1
-      # ThirdEye rule type
-      type: PERCENTAGE_RULE
-      # parameters for this rule
-      params:
-        offset: wo1w
-        change: 0.1
-
-  # (Optional) configure the exclusion rule. (Exclude the anomalies you don't
-  # want to see but detected by the detection rule above)
-  filter:
-    - name: filter_rule_1
-      type: ABSOLUTE_CHANGE_FILTER
-      params:
-        threshold: 10000
-
-# configure more rule if you'd like to
-- detection:
-    - name: detection_rule_2
-      type: ABSOLUTE_CHANGE_RULE
-      params:
-        offset: wo1w
-        change: 1000000
-        pattern: UP
+- detection:                      # Eg. Detect anomalies if the week over week change of this metric is more than 10%
+    - name: detection_rule_1      # Give a unique name for this detection rule.
+      type: PERCENTAGE_RULE       # Configure the detection type here. See doc for more details.
+      params:                     # The parameters for this rule. Different rules have different params.
+        offset: wo1w              # Compare current value with last week. (Values supported - wo1w, wo2w, median3w etc)
+        change: 0.1               # The threshold above which you want to be alerted.
```

Comment:
this parameter name has been changed to 'percentageChange'

Reply:
is the change merged?